split-webapp/split/src/pages/Login.js

```diff
@@ -9,106 +8,169 @@ import "../App.css";
 import AccountCircle from "@material-ui/icons/AccountCircle";
 import VpnKey from "@material-ui/icons/VpnKey";
 import InputAdornment from "@material-ui/core/InputAdornment";
+import PropTypes from "prop-types";
 import mainLogo from "./split2.png";
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    "& .MuiTextField-root": {
-      margin: theme.spacing(1),
-      width: 200,
-      background: "white"
-    }
-  },
-  margin: {
-    margin: theme.spacing(1)
+class Login extends Component {
+  constructor(props) {
+    // assign props and give null/ appropriate values to required fields
+    super(props);
+    this.state = {
+      username: "",
+      password: "",
+      failedAuth: false
+    };
   }
-}));
 
-function Login() {
-  const classes = useStyles();
-  return (
-    <React.Fragment key="LoginKey">
-      <CssBaseline />
-      <Grid
-        container
-        spacing={0}
-        direction="column"
-        alignItems="center"
-        justify="center"
-        style={{ minHeight: "100vh" }}
-      >
-        <Grid item xs={0}>
-          <Container fixed justifyContent="center">
-            <Container fixed>
-              <Box
-                component="div"
-                borderRadius={12}
-                className="SignInContainer"
-              >
-                <img
-                  src={mainLogo}
-                  style={{ width: "50%", marginTop: "10%" }}
-                  alt="Main logo for login"
-                />
-                <Box component="div" className="InnerContainer">
-                  <Typography component="h3" className="SignIn">
-                    Sign In
-                  </Typography>
-                  <form className={classes.root} noValidate autoComplete="off">
+  // hadles live changes in the username field
```

Comment:
Pretty obvious code which means you chose good function names! Don't really need the comments (good thing!)